utils/update_dump.py

```diff
@@ -24,7 +24,7 @@ def error(self, *args):
     self.has_error = True
 
   def generate_dump(self, source, dump_dir):
-    cmd = [self.compiler, "--dump-path=%s" % dump_dir, source]
+    cmd = self.compiler.split(" ") + ["--dump-path=%s" % dump_dir, source]
```

Comment:
Not that it makes any real difference, but I’ve got a slight preference for `shlex.split` here.